src/Symfony/Bridge/Doctrine/Middleware/AutoClose/AutoCloseDriver.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Middleware\AutoClose;
+
+use Doctrine\DBAL\Driver;
+use Doctrine\DBAL\Driver\Connection as DriverConnection;
+use Doctrine\DBAL\Driver\Middleware\AbstractDriverMiddleware;
+
+final class AutoCloseDriver extends AbstractDriverMiddleware
+{
+    public function __construct(
+        Driver $driver,
+        private \ArrayObject $connexionExpire,
```

Comment:
sorry @dunglas, using "expiry" was on purpose, that's the correct word here :)
```suggestion
        private \ArrayObject $connexionExpiries,
```

Reply:
oops :)

Reply:
😂